### DISCUSSION TITLE
Hi FrankenPHP Team, I might be wrong but I might have found a typo in the docs of worker mode of FrankenPHP

### DISCUSSION THREAD

**Discussion by alexandreelise** [php/frankenphp#453] [General]
Hi Kevin,

https://github.com/dunglas/frankenphp/blob/2055142904ec6e6003240873a7ade7dc54e4b6b9/docs/worker.md?plain=1#L83

I might be wrong but it seems that it should be `++$nbRequests <= $_SERVER['MAX_REQUESTS']` because it seems you want to break out of the loop when max requests is reached. But I might miss something.

**Comment by nickchomey** [comment]
I think you might be right. Moreover, am I wrong to think that the 2nd clause 
`!(isset($_SERVER['MAX_REQUESTS'])` makes the 3rd one redundant/impossible? If max requests isn't set, how could the `<=` check even be evaluated?

But perhaps I, too, am missing something.

**Reply by dunglas** [reply]
Good catch, would you mind opening a PR to fix the docs?

**Reply by alexandreelise** [reply]
@dunglas No problem Kevin, I wanted to discuss it first as I might be wrong and didn't want to be a hassle. Doing the PR now.

**Reply by alexandreelise** [reply]
Hi Kevin,
Pull request is done. While I was at it I found something that seems to be a "defect" in the ci. Just a missing METADATA key leading to null image hence failing the amd64 binary build step.
the METADATA key missing seems to be

```sh

'."static-builder"."containerimage.config.digest"'


```

**Reply by alexandreelise** [reply]
Wow! Nice one @nickchomey,

That's why code review and pair programming is better to look at the code with many different views and eyes!

Thanks! I fixed it in the PR.
